resources/css/devbox.css

```diff
@@ -24,6 +24,15 @@
   box-sizing: border-box;
 }
 
+.quotedtext {
+  border-left: 3px solid var(--text-color-muted);
+  padding: 4px 6px 5px 8px;
+  margin: 3px 0px;
+  background: var(--box-bg-color);
+  width: 100%;
+  display: inline-block;
+}
```

Comment:
There's a minor performance benefit to putting this within the `@layer components` block in _resources/css/utilities.css_, namely, we can reach for Tailwind's sane defaults and benefit from the Tailwind compiler's dead code elimination ([docs](https://tailwindcss.com/docs/functions-and-directives)).

I recommend:
```css
@layer components {
  .quotedtext {
    @apply border-l-[3px] border-text-muted;
    @apply pt-1 pb-1.5 pr-1.5 pl-2;
    @apply my-2;
    @apply bg-box-bg;
    @apply w-full;
    @apply inline-block;
  }
```